CIP-0008/README.md

```diff
@@ -388,26 +289,153 @@ We recommend usage of unprotected headers vs protected headers when possible. Th
 
 The public key SHOULD NOT contain any chaincode information, as it could compromise child non-hardened keys. It is both a privacy and a security risk (see [here](https://bitcoin.org/en/wallets-guide#hierarchical-deterministic-key-creation) for more detail).
 
-### Reference implementation
+## Rationale: how does this CIP achieve its goals?
 
-- [COSE-JS](https://github.com/erdtman/cose-js)
-- [Rust message signing](https://github.com/Emurgo/message-signing)
+### Requirements
 
-### Unresolved
+On top being usable for all cases mentioned in *Motivation*, we also desire the following to ensure it works well with hardware wallets:
```

Comment:
```suggestion
On top being usable for all cases mentioned in [Motivation](#motivation), we also desire the following to ensure it works well with hardware wallets:
```

Reply:
@Ryun1 good UX suggestion, I'll have to keep eyes open for more of these 🧐

Reply:
... but unfortunately @Ryun1 @Crypto2099 it doesn't work now that we are using the OMG-it's-long titles:
https://github.com/rphair/CIPs/blob/remediate-0008/CIP-0008/README.md#motivation
... and we'll have to consider whether it's worth linking all of them like this:
https://github.com/rphair/CIPs/blob/remediate-0008/CIP-0008/README.md#motivation-why-is-this-cip-necessary

Reply:
ahh good point!

we probably dont need to enforce it, but I would consider this linking a general nice to have that authors should consider